src/app/components/service/service.component.ts

```diff
@@ -131,26 +132,28 @@ export class ServiceComponent implements OnInit, OnDestroy {
 
     this.subscription.add(
       combineLatest([
-        this.form.controls.status.valueChanges.pipe(startWith(undefined)),
+        this.form.controls.status.valueChanges.pipe(startWith([])),
         this.form.controls.search.valueChanges.pipe(
           startWith(''),
           map((v) => v.toLowerCase()),
         ),
-      ]).subscribe(([status, search]) => {
-        this.filteredTranslations$ = computed(() => {
-          let filtered = !status ? this.translations$() : this.filterMessages(this.translations$(), status.value)
+      ]).subscribe(
+        ([status, search]) =>
+          (this.filteredTranslations$ = computed(() => {
+            let filtered =
+              status?.length === 0 ? this.translations$() : this.filterMessages(this.translations$(), status)
 
-          this.animationState = !status ? 'in' : 'out'
+            this.animationState = status?.length === 0 ? 'in' : 'out'
 
-          this.filtered = status ? true : false
+            this.filtered = status?.length !== 0 ? true : false
```

Comment:
Should `filtered` take into account the `search` value?

Reply:
hm, I just checked and realized that this.filtered is no longer needed. I forgot to remove it from this component

Reply:
but i found another thing to fix. animation when status is changed also works when text search is active